clarify application.summary is an empty string (#4668)
hammerandchisel/discord-api-docs: docs/resources/Application.md

| terms_of_service_url?  | string                                                     | the url of the app's terms of service                                                                                      |
| privacy_policy_url?    | string                                                     | the url of the app's privacy policy                                                                                        |
| owner?                 | partial [user](#DOCS_RESOURCES_USER/user-object) object    | partial user object containing info on the owner of the application                                                        |

> | summary                | string                                                     | **Deprecated** previously if this application is a game sold on Discord, this field will be the summary field for the store page of its primary sku, now an empty string  |

Reviewer: ```suggestion
| summary                | string                                                     | **deprecated**: previously if this application was a game sold on Discord, this field would be the summary field for the store page of its primary SKU; now an empty string  |
```

small grammar nits

Author: thanks
